packages/talker_flutter/lib/src/ui/talker_monitor/talker_monitor.dart

```diff
@@ -21,9 +21,14 @@ class TalkerMonitor extends StatelessWidget {
     return Scaffold(
       backgroundColor: theme.backgroundColor,
       appBar: AppBar(
-        title: const FittedBox(
+        backgroundColor: theme.backgroundColor,
+        iconTheme: IconThemeData(color: theme.textColor),
+        title: FittedBox(
           fit: BoxFit.scaleDown,
-          child: Text('Talker Monitor'),
+          child: Text(
+            'Talker Monitor',
+            style: TextStyle(color: theme.textColor),
+          ),
```

Comment:
This changes isn't related to this PR
Can you create another PR with this changes ?